service/controller/v22/resource/bridgezone/resource.go

```diff
@@ -247,12 +247,10 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 			},
 			HostedZoneId: &intermediateZoneID,
 		}
-		time.Sleep(1 * time.Second)
```

Comment:
We added these back then to address some PM under the premise this resource will not live for too long. With the rewritten implementation we should not hammer the Route53 API anymore too much. So we can get rid of this bullshit.